### PR TITLE
Feature/image list styling

### PR DESCRIPTION
**Pull request by kejmil01**

### CODE REVIEW

File: app/src/test/java/com/invisibleteam/goinvisible/util/ItemDecorationAlbumColumnsTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    private View view;
+
+    @Mock
+    private RecyclerView parent;
+
+    @Mock
+    private RecyclerView.State recyclerViewState;
+
+    @Test
+    public void whenItemDecorationInitializedAndViewInFirstRowAndColumn_spaceBetweenItemCorrect() {
```
Comment (BartSoftware) [inline app/src/test/java/com/invisibleteam/goinvisible/util/ItemDecorationAlbumColumnsTest.java]: I see we have two different tests naming convention
For me it is:
whenItemDecorationInitializedAndViewInFirstRowAndColumn_SpaceBetweenItemCorrect

Comment (BartSoftware) [inline app/src/test/java/com/invisibleteam/goinvisible/util/ItemDecorationAlbumColumnsTest.java]: I see we have different tests naming convention

Comment (kejmil01) [reply app/src/test/java/com/invisibleteam/goinvisible/util/ItemDecorationAlbumColumnsTest.java]: Ok to me, will change to ...SpaceBetweenItemCorrect

Comment (kejmil01) [reply app/src/test/java/com/invisibleteam/goinvisible/util/ItemDecorationAlbumColumnsTest.java]: Done

**Review by michal1990 — APPROVED** [review]

**Pull request merged**